src/validator/MatchValidator.cpp

```diff
@@ -450,6 +450,9 @@ Status MatchValidator::validateWith(const WithClause *with,
     std::vector<const Expression *> exprs;
     exprs.reserve(with->columns()->size());
     for (auto *col : with->columns()->columns()) {
+        if (col->expr()->kind() == Expression::Kind::kLabel) {
+            col->setAlias(new std::string(col->expr()->toString()));
+        }
         if (col->alias() == nullptr) {
```

Comment:
I think you could remove this check.

Reply:
Just add an alias for `Label Expression` implicitly. So this check is necessary